### PR TITLE
Feature/#297 covert unsolicited cn

### PR DESCRIPTION
**Pull request by lkolacz**

### CODE REVIEW

File: backend/unpp_api/apps/project/urls.py [hunk 1]
```diff
@@ -43,6 +44,7 @@
     url(r'^direct/$', DirectProjectAPIView.as_view(), name="direct"),
     url(r'^pins/$', PinProjectAPIView.as_view(), name="pins"),
     url(r'^unsolicited/$', UnsolicitedProjectAPIView.as_view(), name="unsolicited"),
+    url(r'^convert-unsolicited/(?P<pk>\d+)/$', ConvertUnsolicitedAPIView.as_view(), name="convert-unsolicited"),
```
Comment (elyk) [inline backend/unpp_api/apps/project/urls.py]: I think better to reverse: 

/(?P<pk>\d+)/convert-unsolicited/ 

as to not confuse the frontend here. It shows an operation on that resource.

Comment (lkolacz) [reply backend/unpp_api/apps/project/urls.py]: If `pk` will be a project key I agree, but here `pk` is the primary key to application (unsolicited) object  .. that can bring "misunderstandings" IMO.
What is the final word? Do we still want to change this?

Comment (elyk) [reply backend/unpp_api/apps/project/urls.py]: Ok so how about:

`application/(?P<pk>\d+)/convert-unsolicited/`

?

Comment (lkolacz) [reply backend/unpp_api/apps/project/urls.py]: LGTM, will do that!

Comment (elyk) [reply backend/unpp_api/apps/project/urls.py]: Cool. Just add a get_object_or_404 at the view to check it is actually an unsolicited and hasn't been converted. 

application = get_object_or_404(Application, id=self.kwargs['id'], is_unsolicited=True, eoi_converted__isnull=True)

Comment (lkolacz) [reply backend/unpp_api/apps/project/urls.py]: @elyk DONE

**Comment by lkolacz** [comment]
I have write permission to merge this PR from Kyle via slack.

**Pull request merged**